pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -251,63 +259,72 @@ task train_classifier {
 
         qiime feature-classifier extract-reads\
         --i-sequeunces "~{otu_basename}_seqs.qza"\
-        --p-f-primer "~{forward_adapter}"\
-        --p-r-primer "~reverse_adapter}"\
-        ~{"--p-min-length" + min_length}\
-        ~{"--p-max-length" + max_length}\
+        --p-f-primer "~{forward_adapter}" \
+        --p-r-primer "~{reverse_adapter}" \
+        ~{"--p-min-length" + min_length} \
+        ~{"--p-max-length" + max_length} \
         --o-reads "~{otu_basename}_v1-2-ref-seqs.qza"
-#might have to be broken down into two tasks
-        qiime feature-classifier fit-classifier-naive-bayes\ 
-        --i-reference-reads "~{otu_basename}_v1-2-ref-seqs.qza"\ 
-        --i-reference-taxonomy "~{otu_basename}_tax.qza"\ 
+
+        qiime feature-classifier fit-classifier-naive-bayes \ 
+        --i-reference-reads "~{otu_basename}_v1-2-ref-seqs.qza" \ 
+        --i-reference-taxonomy "~{otu_basename}_tax.qza" \ 
         --o-classifier "~{otu_basename}_v1-2-classifier.qza"
+        >>>
     output {
         File    trained_classifier = "~{otu_basename}_v1-2-classifier.qza"
     }
     runtime {
-        docker: "~{docker}"
-        memory: "10 GB"
-        cpu: 1
+        docker: docker
+        memory: "${memory_mb} MiB"
+        cpu: cpu
+        disks: "local-disk ${disk_size_gb} SSD"
     }
-
 }
-train tax_analysis {
+task tax_analysis {
     meta {
         description: "Protocol describes performing a taxonomic classification with a naive bayes classifier that has been trained on the V1-2 regions amplified by our primers."
     }
     input {
-        File trained_classifier
-        File rep-seqs_outfile
-        File rep-table_outfile 
+        File    trained_classifier
+        File    rep_seqs_outfile
+        File    rep_table_outfile 
         String  basename  =   basename(trained_classifier, '.qza')
```

Comment:
I think things get funny (but only sometimes in some cases) when you name a WDL variable a reserved word/WDL function (like `basename`).
```suggestion
        String  classifier_basename  =   basename(trained_classifier, '.qza')
```